### ISSUE TITLE
Add meaningful alts to linked images on About page

### ISSUE THREAD

**Issue by jasonwebb** [jasonwebb/jasonwebb.io-theme#11]
The "highlighted on ..." logo links have only images as inner content but they have no alt text so the links have no label for screen readers.

**Comment by jasonwebb** [comment]
Resolved.
